kick.go

```diff
@@ -0,0 +1,24 @@
+package pitaya
+
+import (
+	"context"
+
+	"github.com/topfreegames/pitaya/logger"
+	"github.com/topfreegames/pitaya/protos"
+	"github.com/topfreegames/pitaya/session"
+)
+
+// SendKickToUser sends kick to an user
+func SendKickToUser(uid, frontendType string) error {
```

Comment:
It would be nice to have a function that accepts a slice of uids to kick as well

Reply:
Multiples uids from multiples servers or multiples uids from one server ? (the last is what we have on push.go)